HPXMLtoOpenStudio/resources/data/schedules_affected.csv

```diff
@@ -0,0 +1,36 @@
+Schedule Name,Affected By Vacancy,Affected By Power Outage
+occupants,true,false
+lighting_interior,true,true
+lighting_exterior,true,true
+lighting_garage,true,true
+lighting_exterior_holiday,true,true
+cooking_range,true,true
+refrigerator,false,true
+extra_refrigerator,false,true
+freezer,true,true
+dishwasher,true,true
+clothes_washer,true,true
+clothes_dryer,true,true
+ceiling_fan,true,true
+plug_loads_other,true,true
+plug_loads_tv,true,true
+plug_loads_vehicle,true,true
+plug_loads_well_pump,true,true
+fuel_loads_grill,true,true
+fuel_loads_lighting,true,true
+fuel_loads_fireplace,true,true
+pool_pump,false,true
+pool_heater,false,true
+hot_tub_pump,false,true
+hot_tub_heater,false,true
+hot_water_dishwasher,true,true
+hot_water_clothes_washer,true,true
+hot_water_fixtures,true,true
+sleeping,false,false
```

Comment:
Why is there a `sleeping` row?

Reply:
Hmm, I'm not sure. It was in the previous "unaffected by vacancy" array. I'll try removing it.